ComponentKit/Components/CKButtonComponent.h

```diff
@@ -18,6 +18,22 @@ struct CKButtonComponentAccessibilityConfiguration {
   NSString *accessibilityLabel;
 };
 
+struct CKButtonTitle {
+  CKButtonTitle() : title(nil), attributedTitle(nil) {};
+  CKButtonTitle(std::nullptr_t v) : title(nil), attributedTitle(nil) {};
+  CKButtonTitle(NSString *t) : title(t), attributedTitle(nil) {};
+  CKButtonTitle(NSAttributedString *t) : title(nil), attributedTitle(t) {};
+
+  NSString *title;
```

Comment:
nit: Since title is already in the name of the struct, I would tend to name these after their types, so string and attributedString. A little less redundant I think? When I see struct.title.title, I'd have to double-check that the type of title is a string...